server/CMakeLists.txt

```diff
@@ -47,6 +51,10 @@ etn_target(static ${PROJECT_NAME}-static
         crypto
         uuid
         yaml-cpp
+        libfty-asset
+        fty-asset-libng
+        tntdb
+        cxxtools
```

Comment:
why tntdb and cxxtools? :(

Reply:
fty-asset-libng need tntdb and cxxtools ? otherwise fty-discovery-ng doesn't compile...